src/murmur/main.cpp

```diff
@@ -285,153 +387,87 @@ int main(int argc, char **argv) {
 
 	qInstallMessageHandler(murmurMessageOutputWithContext);
 
-#ifdef Q_OS_WIN
-	Tray tray(nullptr, &le);
-#endif
+	auto cli_options = parseCLI(argc, argv);
```

Comment:
Please don't use `auto` in a context where the type of the variable is not already visible in the same line.
The only exception we make from this rule is for iterators.

Reply:
I'm not sure how I can get rid of auto in this place, since its returning an unnamed struct. Removing auto would mean to pull the struct out into global space. If you prefer that tho, I can def do that

Reply:
Yeah, I would prefer putting the struct definition in the global namespace and make `parse_CLI` return that type explicitly :thinking: